packages/orbit-components/src/Seat/index.js

```diff
@@ -61,20 +100,27 @@ const Seat = ({
 
   return (
     <Stack inline grow={false} spacing="XXXSmall" direction="column" align="center">
-      <StyledSeat
-        viewBox={resolveViewBoxSize(size, selected)}
-        aria-labelledby={`${titleId} ${descrId}`}
-        size={size}
-        fill="none"
-        role="img"
-        selected={selected}
-        data-test={dataTest}
-      >
-        <title id={titleId}>{title}</title>
-        <desc id={descrId}>{description}</desc>
-        <SeatItem type={type} selected={selected} size={size} price={price} label={label} />
-      </StyledSeat>
-      {price && !(selected && type === TYPES.UNAVAILABLE) && size !== SIZE_OPTIONS.EXTRASMALL && (
+      <StyledSeatWrapper onClick={onClick} type={type} selected={selected} tabIndex={0}>
```

Comment:
I missed this detail before, but If I understood correctly, unavailable seats should not be focusable, so `tabIndex` value should be conditional.